### PR TITLE
[Chore](build) fix compile failed on SegmentFlusher::_expand_variant_to_subcolumns

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes
1. fix compile failed on SegmentFlusher::_expand_variant_to_subcolumns
2. revert #32388
## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by eldenmoon — DISMISSED** [review]
LGTM

**Comment by BiteTheDDDDt** [comment]
run buildall

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -2248,6 +2248,9 @@ public class Config extends ConfigBase {
     @ConfField(mutable = false)
     public static boolean allow_analyze_statistics_info_polluting_file_cache = true;
 
+    @ConfField
+    public static int cpu_resource_limit_per_analyze_task = 1;
```
Comment (morrySnow) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:2252]: why into new feature in a fix compile PR?

**Review by zclllyybb — APPROVED** [review]

**Review by morningman — APPROVED** [review]

**Review by Gabriel39 — APPROVED** [review]

**Pull request merged**